es-ds-components/.eslintrc.json

```diff
@@ -0,0 +1,133 @@
+{
+    "parser": "vue-eslint-parser",
+    "parserOptions": {
+        "parser": "@typescript-eslint/parser"
+    },
+    "settings": {
+        "import/resolver": {
+            "typescript": {
+                "alwaysTryTypes": true
+            },
+            "alias": {
+                "map": [
+                    ["@", "./lib"],
+                    ["~", "."]
+                ]
+            }
+        }
+    },
+    "plugins": ["@typescript-eslint"],
+    "env": {
+        "node": true
+    },
+    "overrides": [
+        {
+            "files": ["*.yaml", "*.yml"],
+            "parser": "yaml-eslint-parser",
+            "rules": {
+                "max-len": "off"
+            }
+        }
+    ],
+    "extends": [
```

Comment:
I wonder if we could start with something like:

```
  "extends": [
    "eslint:recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:vue/vue3-recommended",
    "prettier"
  ],
```

Per the docs https://eslint.vuejs.org/user-guide/#configuration-eslint-config-js